static/authorityConfig.json

```diff
@@ -24,6 +24,30 @@
     "language": "en",
     "component": "lookup"
   },
+  {
+    "label": "CERL_person (QA)",

```

Comment:
Do you want the _ in the labels for CERL?

Reply:
Fixed this.  And fixed it for geonames too.